manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -295,6 +295,7 @@ metadata:
     {{ toJsonMap (omit .InfrastructureAnnotations "kubectl.kubernetes.io/last-applied-configuration" "gateway.istio.io/name-override" "gateway.istio.io/service-account" "gateway.istio.io/controller-version") | nindent 4 }}
   labels:
     {{- toJsonMap
+      (strdict "experimental.istio.io/traffic-distribution" "PreferClose")
```

Comment:
I used to think that all annotations is experimental before promote to offical API.